Lib/test/test_email/test_email.py

```diff
@@ -3313,6 +3316,78 @@ def test_parsing_errors(self):
         # Test email.utils.supports_strict_parsing attribute
         self.assertEqual(email.utils.supports_strict_parsing, True)
 
+    def test_parsing_errors_strict_disabled_via_env_var(self):
+        address = 'alice@example.org )Alice('
+        empty = ('', '')
+
+        # Delete cached default value to make the function
+        # reload the environment variable provided below.
+        try:
+            del utils._cached_strict_addr_parsing
+        except AttributeError:
+            pass
+
+        # Strict disabled via env variable, old behavior expected
+        with EnvironmentVarGuard() as environ:
+            environ["PYTHON_EMAIL_DISABLE_STRICT_ADDR_PARSING"] = "1"
```

Comment:
Can you add an test where the env var is set to an empty string?

Reply:
Sure thing.

Reply:
Implemented in the first fixup commit.